src/style.css

```diff
@@ -0,0 +1,181 @@
+html {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  max-width: 198px;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.photo {
+  height: 134px;
+  width: 160px;
```

Comment:
this only makes sense in the context of the card. maybe move to the .card__photo selector?